package.json

```diff
@@ -12,6 +12,7 @@
         "analyze": "lit-analyzer \"{packages,tools}/*/src/**/!(*.css).ts\"",
         "build": "wireit",
         "build:clear-cache": "rimraf packages/*/tsconfig.tsbuildinfo && rimraf tools/*/tsconfig.tsbuildinfo",
+        "build:component:css": "node ./tasks/process-spectrum.js",
```

Comment:
I wanted the ability to force-run the build only for specific components as a means of validating updates to the spectrum-config files.